### PR TITLE
Feature/122 time series data

### PR DESCRIPTION
**Pull request by mohitb3**
- Begin work on data conversion of RADAR time series data #122
- Time Series data plots #135
- fixed bug with search box on homepage
- update LocationPage to use useEffect to fetch the location detail data; this ensures that if the redux bundler contains location detail data for a previous location, a new fetch call will be made

### CODE REVIEW

File: src/app-bundles/radar-time-series-bundle.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import createRestBundle from "./create-rest-bundle";
-import { getRestUrl } from "./bundle-utils";
 import { createSelector } from "redux-bundler";
-import { LOCATIONDETAIL_SET_CODE } from "./location-detail-bundle";
+import createRestBundle from "./create-rest-bundle";
+import { getRestUrl, getIntervalTime } from "./bundle-utils";
```
Comment (dlin2102) [inline src/app-bundles/radar-time-series-bundle.js:3]: Hmm on my branch when I pulled down, I don't see the getIntervalTime func in bundle-utils. Did it not get pushed up?

Comment (mohitb3) [reply src/app-bundles/radar-time-series-bundle.js:3]: getIntervalTime isn't part of the bundle-utils, it comes with the createNestedUrlBundle which is part of cormaps. It confused me at first too

Comment (mohitb3) [reply src/app-bundles/radar-time-series-bundle.js:3]: ahh yea sorry, forgot to push it up

Comment (mohitb3) [reply src/app-bundles/radar-time-series-bundle.js:3]: It should be pushed up now

**Review by dlin2102 — APPROVED** [review]

**Comment by brian428** [comment]
@mohitb3 I pushed a couple of small tweaks if you want to take a look.

**Comment by mohitb3** [comment]
@brian428 looks good to me

**Pull request merged**